CI: MSYS2 Clang 20: ../../src/Glitch64/OGLglitchmain.cpp:1219:7: error: no matching function for call to 'grSstWinOpen'

Current CI: https://github.com/mupen64plus/mupen64plus-video-glide64mk2/actions/runs/14782085179

```
../../src/Glitch64/OGLglitchmain.cpp:1219:7: error: no matching function for call to 'grSstWinOpen'
 1219 |       grSstWinOpen((unsigned long)NULL, GR_RESOLUTION_640x480 | 0x80000000, 0, GR_COLORFORMAT_ARGB,
      |       ^~~~~~~~~~~~
../../src/Glitch64/OGLglitchmain.cpp:454:1: note: candidate function not viable: no known conversion from 'unsigned long' to 'HWND' (aka 'HWND__ *') for 1st argument
  454 | grSstWinOpen(
      | ^
  455 |              HWND                 hWnd,
      |              ~~~~~~~~~~~~~~~~~~~~~~~~~
../../src/Glitch64/OGLglitchmain.cpp:1317:52: warning: format specifies type 'unsigned int' but the argument has type 'FxU32' (aka 'unsigned long') [-Wformat]
 1317 |     display_warning("unknown pname in grGet : %x", pname);
      |                                               ~~   ^~~~~
      |                                               %lx
../../src/Glitch64/OGLglitchmain.cpp:1359:58: warning: format specifies type 'unsigned int' but the argument has type 'FxU32' (aka 'unsigned long') [-Wformat]
 1359 |     display_warning("unknown grGetString selector : %x", pname);
      |                                                     ~~   ^~~~~
      |                                                     %lx
../../src/Glitch64/OGLglitchmain.cpp:1617:61: warning: format specifies type 'unsigned int' but the argument has type 'GrBuffer_t' (aka 'long') [-Wformat]
 1617 |     display_warning("grRenderBuffer : unknown buffer : %x", buffer);
      |                                                        ~~   ^~~~~~
      |                                                        %lx
../../src/Glitch64/OGLglitchmain.cpp:1509:17: warning: unused variable 'region' [-Wunused-variable]
 1509 |   static HANDLE region = NULL;
      |                 ^~~~~~
../../src/Glitch64/OGLglitchmain.cpp:1510:7: warning: unused variable 'realWidth' [-Wunused-variable]
 1510 |   int realWidth = pBufferWidth, realHeight = pBufferHeight;
      |       ^~~~~~~~~
../../src/Glitch64/OGLglitchmain.cpp:1510:33: warning: unused variable 'realHeight' [-Wunused-variable]
 1510 |   int realWidth = pBufferWidth, realHeight = pBufferHeight;
      |                                 ^~~~~~~~~~
../../src/Glitch64/OGLglitchmain.cpp:1756:58: warning: format specifies type 'unsigned int' but the argument has type 'GrBuffer_t' (aka 'long') [-Wformat]
 1756 |       display_warning("grLfbLock : unknown buffer : %x", buffer);
      |                                                     ~~   ^~~~~~
      |                                                     %lx
../../src/Glitch64/OGLglitchmain.cpp:1838:59: warning: format specifies type 'unsigned int' but the argument has type 'GrBuffer_t' (aka 'long') [-Wformat]
 1838 |     display_warning("grReadRegion : unknown buffer : %x", src_buffer);
      |                                                      ~~   ^~~~~~~~~~
      |                                                      %lx
../../src/Glitch64/OGLglitchmain.cpp:1908:63: warning: format specifies type 'unsigned int' but the argument has type 'GrBuffer_t' (aka 'long') [-Wformat]
 1908 |     display_warning("grLfbWriteRegion : unknown buffer : %x", dst_buffer);
      |                                                          ~~   ^~~~~~~~~~
      |                                                          %lx
../../src/Glitch64/OGLglitchmain.cpp:1961:65: warning: format specifies type 'int' but the argument has type 'GrLfbSrcFmt_t' (aka 'unsigned long') [-Wformat]
 1961 |       display_warning("grLfbWriteRegion : unknown format : %d", src_format);
      |                                                            ~~   ^~~~~~~~~~
      |                                                            %lu
../../src/Glitch64/OGLglitchmain.cpp:1994:63: warning: format specifies type 'unsigned int' but the argument has type 'GrLfbSrcFmt_t' (aka 'unsigned long') [-Wformat]
 1994 |       display_warning("unknown depth buffer write format:%x", src_format);
      |                                                          ~~   ^~~~~~~~~~
      |                                                          %lx
../../src/Glitch64/OGLglitchmain.cpp:1997:46: warning: format specifies type 'int' but the argument has type 'FxU32' (aka 'unsigned long') [-Wformat]
 1997 |       display_warning("dst_x:%d, dst_y:%d\n",dst_x, dst_y);
      |                              ~~              ^~~~~
      |                              %lu
../../src/Glitch64/OGLglitchmain.cpp:1997:53: warning: format specifies type 'int' but the argument has type 'FxU32' (aka 'unsigned long') [-Wformat]
 1997 |       display_warning("dst_x:%d, dst_y:%d\n",dst_x, dst_y);
      |                                        ~~           ^~~~~
      |                                        %lu
15 warnings and 1 error generated.
make: *** [Makefile:531: _obj/Glitch64/OGLglitchmain.o] Error 1
make: *** Waiting for unfinished jobs....
../../src/GlideHQ/TxCache.cpp:322:89: warning: format specifies type 'char *' but the argument has type 'wchar_t *' [-Wformat]
  322 |         ERRLOG("Error while changing current directory back to original path of '%s'!", curpath);
      |                                                                                  ~~     ^~~~~~~
      |                                                                                  %ls
../../src/GlideHQ/../Glide64/Gfx_1.3.h:213:44: note: expanded from macro 'ERRLOG'
  213 | #define ERRLOG(...) WriteLog(M64MSG_ERROR, __VA_ARGS__)
      |                                            ^~~~~~~~~~~
../../src/GlideHQ/TxCache.cpp:490:87: warning: format specifies type 'char *' but the argument has type 'wchar_t *' [-Wformat]
  490 |       ERRLOG("Error while changing current directory back to original path of '%s'!", curpath);
      |                                                                                ~~     ^~~~~~~
      |                                                                                %ls
../../src/GlideHQ/../Glide64/Gfx_1.3.h:213:44: note: expanded from macro 'ERRLOG'
  213 | #define ERRLOG(...) WriteLog(M64MSG_ERROR, __VA_ARGS__)
      |                                            ^~~~~~~~~~~
2 warnings generated.
make: Leaving directory '/d/a/mupen64plus-video-glide64mk2/mupen64plus-video-glide64mk2/projects/unix'
Error: Process completed with exit code 2.
```

--- a/src/Glitch64/OGLESglitchmain.cpp
+++ b/src/Glitch64/OGLESglitchmain.cpp
@@ -1209,7 +1209,7 @@ grGet( FxU32 pname, FxU32 plength, FxI32 *params )
     if (plength < 4 || params == NULL) return 0;
     if (!nbTextureUnits)
     {
-      grSstWinOpen((unsigned long)NULL, GR_RESOLUTION_640x480 | 0x80000000, 0, GR_COLORFORMAT_ARGB,
+      grSstWinOpen(NULL, GR_RESOLUTION_640x480 | 0x80000000, 0, GR_COLORFORMAT_ARGB,
         GR_ORIGIN_UPPER_LEFT, 2, 1);
       grSstWinClose(0);
     }

--- a/src/Glitch64/OGLglitchmain.cpp
+++ b/src/Glitch64/OGLglitchmain.cpp
@@ -1216,7 +1216,7 @@ grGet( FxU32 pname, FxU32 plength, FxI32 *params )
     if (plength < 4 || params == NULL) return 0;
     if (!nbTextureUnits)
     {
-      grSstWinOpen((unsigned long)NULL, GR_RESOLUTION_640x480 | 0x80000000, 0, GR_COLORFORMAT_ARGB,
+      grSstWinOpen(NULL, GR_RESOLUTION_640x480 | 0x80000000, 0, GR_COLORFORMAT_ARGB,
         GR_ORIGIN_UPPER_LEFT, 2, 1);
       grSstWinClose(0);
     }

